### DISCUSSION TITLE
Configure navigation links

### DISCUSSION THREAD

**Discussion by tripleo1** [squidfunk/mkdocs-material#2776] [Q&A] [answered]
How do I set up my navigation sidebar so its not alphabetical? 

And how do I put them in a hierarchy?

**Comment by tripleo1** [comment, marked as answer]
I found it! It's nav in mkdocs.yml.  

It would be nice to see this in the documentation somewhere?

Have you considered a SUMMARY.md like I think mdbook does it?  I'm not saying I'm looking for a change, just asking...

**Reply by squidfunk** [reply]
Thanks for suggesting. https://www.mkdocs.org/ already explains this. It's best to consult their documentation for everything outside the `theme` and `extra` variables. Yes, there are some variables which we also explain in our docs, like for example `repo_url`, but this is only due to the immediate effects it has on the theme. We won't be copying everything from their docs.

**Reply by tripleo1** [reply]
Thanks for your positive reply. I forgot you're just a "theme". And thanks for the advice, you helped solve another problem I was about to write a ticket on.
